api/api.go

```diff
@@ -54,6 +54,17 @@ type CreateNamespaceResponse struct {
 	GQLResponseErrors
 }
 
+// CreateOrbResponse type matches the data shape of the GQL response for
+// creating an orb
+type CreateOrbResponse struct {
+	Orb struct {
+		CreatedAt string
```

Comment:
`createdAt` doesn't seem to be used anywhere. Let's remove from the GQL query and the response struct.